python/pyarrow/ipc.pxi

```diff
@@ -757,7 +757,7 @@ cdef class RecordBatchReader(_Weakrefable):
         return self
 
     @staticmethod
-    def from_batches(schema, batches):
+    def from_batches(Schema schema, batches):
```

Comment:
I don't think this protects `schema` from being None, though?

Reply:
Ah, yes, will add a `not None`